### PR TITLE
Add/newsletter preview menu item

### PR DESCRIPTION
**Pull request by lezama**
Fixes #

## Proposed changes:

* Add an Email Preview item to the Preview Menu

Screenshot 2024-10-16 at 3 59 03 PM

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to the editor and check that the ‘Preview Email’ menu appears. When you click it, the preview modal should open.
* Please check that there are no regressions in the Newsletter menu.

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/subscriptions/editor.js [hunk 1]
```diff
@@ -71,17 +75,58 @@ const shouldShowNewsletterMenu = () => {
...
+		<>
+			<SubscribePanels />
+			{ shouldShowNewsletterMenu() && (
+				<>
+					{ PluginPreviewMenuItem ? (
+						<PluginPreviewMenuItem
+							onClick={ () => openPreviewModal( 'preview_menu' ) }
+							icon={ send }
+						>
+							{ __( 'Email Preview', 'jetpack' ) }
```
Comment (simison) [inline projects/plugins/jetpack/extensions/blocks/subscriptions/editor.js]: Sentence case "Email preview"

**Comment by crisbusquets** [comment]
@lezama I can't see the panel in a Jurassic Ninja I created 😞 

But from what I see in your screenshot, the items are misaligned:
377194520-3834ff38-4247-4ab1-ac27-af01226b0def

**Comment by lezama** [comment]
> But from what I see in your screenshot, the items are misaligned:

This is coming from core. @simison I thought you opened an issue but I couldn’t find it.

**Comment by simison** [comment]
> But from what I see in your screenshot, the items are misaligned:

Known issue in core:  https://github.com/WordPress/gutenberg/pull/64644#issuecomment-2304783018

**Review by simison — APPROVED** [review]
Looks good and tests well for me.

I didn't get to test scenario when the module is disabled; could you make sure we don't add the preview item in that case before merging?

**Comment by lezama** [comment]
> could you make sure we don't add the preview item in that case before merging?

confirmed.

**Pull request merged**